failing test (nose2.tests.functional.test_coverage.TestCoverage) in 0.5.0

From what I can tell the output expected for coverage in the following test no longer matches expectations (I'm probably just not fully aware of what's happening).  I'm working on packaging this for Gentoo and could be missing a dependency as well.  Any guidance would be appreciated.

Failing test:

```
======================================================================
FAIL: test_run (nose2.tests.functional.test_coverage.TestCoverage)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "./nose2/tests/functional/test_coverage.py", line 19, in test_run
    stderr=os.path.join('lib', 'mod1').replace('\\', r'\\') + STATS)
  File "./nose2/tests/_common.py", line 64, in assertTestRunOutputMatches
    testf(util.safe_decode(cmd_stderr), stderr)
AssertionError: Regex didn't match: 'lib/mod1           8      5    38%' not found in 'test1 (test_coverage.TestLib) ... ok\n\n----------------------------------------------------------------------\nRan 1 test in 0.001s\n\nOK\n'

-------------------- >> begin captured logging << --------------------
nose2.main: DEBUG: logging initialized 30
nose2.session: DEBUG: Loading plugin modules: {'nose2.plugins.logcapture', 'nose2.plugins.result', 'nose2.plugins.loader.generators', 'nose2.plugins.loader.testclasses', 'nose2.plugins.buffer', 'nose2.plugins.loader.discovery', 'nose2.plugins.loader.testcases', 'nose2.plugins.coverage', 'nose2.plugins.loader.functions', 'nose2.plugins.failfast', 'nose2.plugins.loader.loadtests', 'nose2.plugins.loader.parameters', 'nose2.plugins.debugger', 'nose2.plugins.dundertest'}
nose2.session: DEBUG: Plugin is available: <class 'nose2.plugins.logcapture.LogCapture'>
nose2.session: DEBUG: Plugin is available: <class 'nose2.events.Plugin'>
nose2.session: DEBUG: Plugin is available: <class 'nose2.plugins.result.ResultReporter'>
nose2.session: DEBUG: Register active plugin <nose2.plugins.result.ResultReporter object at 0x7fb780bd11d0>
nose2.session: DEBUG: Register method startTest for plugin <nose2.plugins.result.ResultReporter object at 0x7fb780bd11d0>
nose2.session: DEBUG: Register method testOutcome for plugin <nose2.plugins.result.ResultReporter object at 0x7fb780bd11d0>
nose2.session: DEBUG: Register method wasSuccessful for plugin <nose2.plugins.result.ResultReporter object at 0x7fb780bd11d0>
nose2.session: DEBUG: Register method afterTestRun for plugin <nose2.plugins.result.ResultReporter object at 0x7fb780bd11d0>
nose2.session: DEBUG: Plugin is available: <class 'nose2.plugins.loader.generators.Generators'>
nose2.session: DEBUG: Register active plugin <nose2.plugins.loader.generators.Generators object at 0x7fb7833fa3d0>
nose2.session: DEBUG: Register method loadTestsFromModule for plugin <nose2.plugins.loader.generators.Generators object at 0x7fb7833fa3d0>
nose2.session: DEBUG: Register method loadTestsFromName for plugin <nose2.plugins.loader.generators.Generators object at 0x7fb7833fa3d0>
nose2.session: DEBUG: Register method loadTestsFromTestCase for plugin <nose2.plugins.loader.generators.Generators object at 0x7fb7833fa3d0>
nose2.session: DEBUG: Register method getTestCaseNames for plugin <nose2.plugins.loader.generators.Generators object at 0x7fb7833fa3d0>
nose2.session: DEBUG: Plugin is available: <class 'nose2.events.Plugin'>
nose2.session: DEBUG: Plugin is available: <class 'nose2.plugins.loader.testclasses.TestClassLoader'>
nose2.session: DEBUG: Register active plugin <nose2.plugins.loader.testclasses.TestClassLoader object at 0x7fb78057a410>
nose2.session: DEBUG: Register method loadTestsFromModule for plugin <nose2.plugins.loader.testclasses.TestClassLoader object at 0x7fb78057a410>
nose2.session: DEBUG: Register method loadTestsFromName for plugin <nose2.plugins.loader.testclasses.TestClassLoader object at 0x7fb78057a410>
nose2.session: DEBUG: Plugin is available: <class 'nose2.plugins.buffer.OutputBufferPlugin'>
nose2.session: DEBUG: Plugin is available: <class 'nose2.plugins.loader.discovery.DiscoveryLoader'>
nose2.session: DEBUG: Register active plugin <nose2.plugins.loader.discovery.DiscoveryLoader object at 0x7fb78057add0>
nose2.session: DEBUG: Register method loadTestsFromNames for plugin <nose2.plugins.loader.discovery.DiscoveryLoader object at 0x7fb78057add0>
nose2.session: DEBUG: Register method loadTestsFromName for plugin <nose2.plugins.loader.discovery.DiscoveryLoader object at 0x7fb78057add0>
nose2.session: DEBUG: Plugin is available: <class 'nose2.plugins.loader.testcases.TestCaseLoader'>
nose2.session: DEBUG: Register active plugin <nose2.plugins.loader.testcases.TestCaseLoader object at 0x7fb78057ab50>
nose2.session: DEBUG: Register method loadTestsFromModule for plugin <nose2.plugins.loader.testcases.TestCaseLoader object at 0x7fb78057ab50>
nose2.session: DEBUG: Register method loadTestsFromName for plugin <nose2.plugins.loader.testcases.TestCaseLoader object at 0x7fb78057ab50>
nose2.session: DEBUG: Plugin is available: <class 'nose2.plugins.coverage.Coverage'>
nose2.session: DEBUG: Plugin is available: <class 'nose2.events.Plugin'>
nose2.session: DEBUG: Plugin is available: <class 'nose2.plugins.loader.functions.Functions'>
nose2.session: DEBUG: Register active plugin <nose2.plugins.loader.functions.Functions object at 0x7fb78057e690>
nose2.session: DEBUG: Register method loadTestsFromModule for plugin <nose2.plugins.loader.functions.Functions object at 0x7fb78057e690>
nose2.session: DEBUG: Register method loadTestsFromName for plugin <nose2.plugins.loader.functions.Functions object at 0x7fb78057e690>
nose2.session: DEBUG: Plugin is available: <class 'nose2.events.Plugin'>
nose2.session: DEBUG: Plugin is available: <class 'nose2.plugins.failfast.FailFast'>
nose2.session: DEBUG: Plugin is available: <class 'nose2.plugins.loader.loadtests.LoadTestsLoader'>
nose2.session: DEBUG: Register active plugin <nose2.plugins.loader.loadtests.LoadTestsLoader object at 0x7fb78057e750>
nose2.session: DEBUG: Register method moduleLoadedSuite for plugin <nose2.plugins.loader.loadtests.LoadTestsLoader object at 0x7fb78057e750>
nose2.session: DEBUG: Register method handleDir for plugin <nose2.plugins.loader.loadtests.LoadTestsLoader object at 0x7fb78057e750>
nose2.session: DEBUG: Plugin is available: <class 'nose2.plugins.loader.parameters.Parameters'>
nose2.session: DEBUG: Register active plugin <nose2.plugins.loader.parameters.Parameters object at 0x7fb780581090>
nose2.session: DEBUG: Register method loadTestsFromModule for plugin <nose2.plugins.loader.parameters.Parameters object at 0x7fb780581090>
nose2.session: DEBUG: Register method loadTestsFromName for plugin <nose2.plugins.loader.parameters.Parameters object at 0x7fb780581090>
nose2.session: DEBUG: Register method getTestCaseNames for plugin <nose2.plugins.loader.parameters.Parameters object at 0x7fb780581090>
nose2.session: DEBUG: Register method getTestMethodNames for plugin <nose2.plugins.loader.parameters.Parameters object at 0x7fb780581090>
nose2.session: DEBUG: Plugin is available: <class 'nose2.events.Plugin'>
nose2.session: DEBUG: Plugin is available: <class 'nose2.plugins.debugger.Debugger'>
nose2.session: DEBUG: Plugin is available: <class 'nose2.plugins.dundertest.DunderTestFilter'>
nose2.session: DEBUG: Register active plugin <nose2.plugins.dundertest.DunderTestFilter object at 0x7fb780581810>
nose2.session: DEBUG: Register method startTestRun for plugin <nose2.plugins.dundertest.DunderTestFilter object at 0x7fb780581810>
nose2.session: DEBUG: Register active plugin <nose2.plugins.coverage.Coverage object at 0x7fb78057e110>
nose2.session: DEBUG: Register method startTestRun for plugin <nose2.plugins.coverage.Coverage object at 0x7fb78057e110>
nose2.session: DEBUG: Register method afterSummaryReport for plugin <nose2.plugins.coverage.Coverage object at 0x7fb78057e110>
nose2.main: DEBUG: Create tests from []/None
nose2.plugins.loader.discovery: DEBUG: Received event LoadFromNames(names=[], module=None)
nose2.plugins.loader.discovery: DEBUG: _discover in /var/tmp/portage/dev-python/nose2-0.5.0/work/nose2-0.5.0/nose2/tests/functional/support/scenario/test_with_module (/var/tmp/portage/dev-python/nose2-0.5.0/work/nose2-0.5.0/nose2/tests/functional/support/scenario/test_with_module)
nose2.plugins.loader.discovery: DEBUG: _find_tests('/var/tmp/portage/dev-python/nose2-0.5.0/work/nose2-0.5.0/nose2/tests/functional/support/scenario/test_with_module', '/var/tmp/portage/dev-python/nose2-0.5.0/work/nose2-0.5.0/nose2/tests/functional/support/scenario/test_with_module')
nose2.plugins.loader.discovery: DEBUG: find in dir /var/tmp/portage/dev-python/nose2-0.5.0/work/nose2-0.5.0/nose2/tests/functional/support/scenario/test_with_module (/var/tmp/portage/dev-python/nose2-0.5.0/work/nose2-0.5.0/nose2/tests/functional/support/scenario/test_with_module)
nose2.plugins.loader.discovery: DEBUG: _find_tests('/var/tmp/portage/dev-python/nose2-0.5.0/work/nose2-0.5.0/nose2/tests/functional/support/scenario/test_with_module/lib', '/var/tmp/portage/dev-python/nose2-0.5.0/work/nose2-0.5.0/nose2/tests/functional/support/scenario/test_with_module')
nose2.plugins.loader.discovery: DEBUG: find in dir /var/tmp/portage/dev-python/nose2-0.5.0/work/nose2-0.5.0/nose2/tests/functional/support/scenario/test_with_module/lib (/var/tmp/portage/dev-python/nose2-0.5.0/work/nose2-0.5.0/nose2/tests/functional/support/scenario/test_with_module)
nose2.plugins.loader.discovery: DEBUG: find in file /var/tmp/portage/dev-python/nose2-0.5.0/work/nose2-0.5.0/nose2/tests/functional/support/scenario/test_with_module/lib/__init__.py (/var/tmp/portage/dev-python/nose2-0.5.0/work/nose2-0.5.0/nose2/tests/functional/support/scenario/test_with_module)
nose2.plugins.loader.discovery: DEBUG: find in file /var/tmp/portage/dev-python/nose2-0.5.0/work/nose2-0.5.0/nose2/tests/functional/support/scenario/test_with_module/lib/mod1.py (/var/tmp/portage/dev-python/nose2-0.5.0/work/nose2-0.5.0/nose2/tests/functional/support/scenario/test_with_module)
nose2.plugins.loader.discovery: DEBUG: find in file /var/tmp/portage/dev-python/nose2-0.5.0/work/nose2-0.5.0/nose2/tests/functional/support/scenario/test_with_module/test_coverage.py (/var/tmp/portage/dev-python/nose2-0.5.0/work/nose2-0.5.0/nose2/tests/functional/support/scenario/test_with_module)
nose2.plugins.loader.generators: DEBUG: loadTestsFromTestCase <class 'test_coverage.TestLib'>
nose2.plugins.loader.generators: DEBUG: getTestCaseNames <class 'test_coverage.TestLib'>
nose2.plugins.loader.parameters: DEBUG: getTestCaseNames GetTestCaseNamesEvent(handled=False, loader=<PluggableTestLoader>, testCase=<class 'test_coverage.TestLib'>, testMethodPrefix=None, extraNames=[], excludedNames=[], isTestMethod=<function TestCaseLoader._getTestCaseNames.<locals>.isTestMethod at 0x7fb780e8f560>)
nose2.loader: DEBUG: loadTestsFromNames event LoadFromNames(names=[], module=None) result <unittest.suite.TestSuite tests=[<unittest.suite.TestSuite tests=[<unittest.suite.TestSuite tests=[<test_coverage.TestLib testMethod=test1>]>]>]>
nose2.main: DEBUG: logging initialized 30
nose2.session: DEBUG: Loading plugin modules: {'nose2.plugins.logcapture', 'nose2.plugins.result', 'nose2.plugins.loader.generators', 'nose2.plugins.loader.testclasses', 'nose2.plugins.buffer', 'nose2.plugins.loader.discovery', 'nose2.plugins.loader.testcases', 'nose2.plugins.coverage', 'nose2.plugins.loader.functions', 'nose2.plugins.failfast', 'nose2.plugins.loader.loadtests', 'nose2.plugins.loader.parameters', 'nose2.plugins.debugger', 'nose2.plugins.dundertest'}
nose2.session: DEBUG: Plugin is available: <class 'nose2.plugins.logcapture.LogCapture'>
nose2.session: DEBUG: Plugin is available: <class 'nose2.events.Plugin'>
nose2.session: DEBUG: Plugin is available: <class 'nose2.plugins.result.ResultReporter'>
nose2.session: DEBUG: Register active plugin <nose2.plugins.result.ResultReporter object at 0x7fb7805899d0>
nose2.session: DEBUG: Register method startTest for plugin <nose2.plugins.result.ResultReporter object at 0x7fb7805899d0>
nose2.session: DEBUG: Register method testOutcome for plugin <nose2.plugins.result.ResultReporter object at 0x7fb7805899d0>
nose2.session: DEBUG: Register method wasSuccessful for plugin <nose2.plugins.result.ResultReporter object at 0x7fb7805899d0>
nose2.session: DEBUG: Register method afterTestRun for plugin <nose2.plugins.result.ResultReporter object at 0x7fb7805899d0>
nose2.session: DEBUG: Plugin is available: <class 'nose2.plugins.loader.generators.Generators'>
nose2.session: DEBUG: Register active plugin <nose2.plugins.loader.generators.Generators object at 0x7fb780589e50>
nose2.session: DEBUG: Register method loadTestsFromModule for plugin <nose2.plugins.loader.generators.Generators object at 0x7fb780589e50>
nose2.session: DEBUG: Register method loadTestsFromName for plugin <nose2.plugins.loader.generators.Generators object at 0x7fb780589e50>
nose2.session: DEBUG: Register method loadTestsFromTestCase for plugin <nose2.plugins.loader.generators.Generators object at 0x7fb780589e50>
nose2.session: DEBUG: Register method getTestCaseNames for plugin <nose2.plugins.loader.generators.Generators object at 0x7fb780589e50>
nose2.session: DEBUG: Plugin is available: <class 'nose2.events.Plugin'>
nose2.session: DEBUG: Plugin is available: <class 'nose2.plugins.loader.testclasses.TestClassLoader'>
nose2.session: DEBUG: Register active plugin <nose2.plugins.loader.testclasses.TestClassLoader object at 0x7fb78050e450>
nose2.session: DEBUG: Register method loadTestsFromModule for plugin <nose2.plugins.loader.testclasses.TestClassLoader object at 0x7fb78050e450>
nose2.session: DEBUG: Register method loadTestsFromName for plugin <nose2.plugins.loader.testclasses.TestClassLoader object at 0x7fb78050e450>
nose2.session: DEBUG: Plugin is available: <class 'nose2.plugins.buffer.OutputBufferPlugin'>
nose2.session: DEBUG: Plugin is available: <class 'nose2.plugins.loader.discovery.DiscoveryLoader'>
nose2.session: DEBUG: Register active plugin <nose2.plugins.loader.discovery.DiscoveryLoader object at 0x7fb78050e910>
nose2.session: DEBUG: Register method loadTestsFromNames for plugin <nose2.plugins.loader.discovery.DiscoveryLoader object at 0x7fb78050e910>
nose2.session: DEBUG: Register method loadTestsFromName for plugin <nose2.plugins.loader.discovery.DiscoveryLoader object at 0x7fb78050e910>
nose2.session: DEBUG: Plugin is available: <class 'nose2.plugins.loader.testcases.TestCaseLoader'>
nose2.session: DEBUG: Register active plugin <nose2.plugins.loader.testcases.TestCaseLoader object at 0x7fb78050ec10>
nose2.session: DEBUG: Register method loadTestsFromModule for plugin <nose2.plugins.loader.testcases.TestCaseLoader object at 0x7fb78050ec10>
nose2.session: DEBUG: Register method loadTestsFromName for plugin <nose2.plugins.loader.testcases.TestCaseLoader object at 0x7fb78050ec10>
nose2.session: DEBUG: Plugin is available: <class 'nose2.plugins.coverage.Coverage'>
nose2.session: DEBUG: Plugin is available: <class 'nose2.events.Plugin'>
nose2.session: DEBUG: Plugin is available: <class 'nose2.plugins.loader.functions.Functions'>
nose2.session: DEBUG: Register active plugin <nose2.plugins.loader.functions.Functions object at 0x7fb780512250>
nose2.session: DEBUG: Register method loadTestsFromModule for plugin <nose2.plugins.loader.functions.Functions object at 0x7fb780512250>
nose2.session: DEBUG: Register method loadTestsFromName for plugin <nose2.plugins.loader.functions.Functions object at 0x7fb780512250>
nose2.session: DEBUG: Plugin is available: <class 'nose2.events.Plugin'>
nose2.session: DEBUG: Plugin is available: <class 'nose2.plugins.failfast.FailFast'>
nose2.session: DEBUG: Plugin is available: <class 'nose2.plugins.loader.loadtests.LoadTestsLoader'>
nose2.session: DEBUG: Register active plugin <nose2.plugins.loader.loadtests.LoadTestsLoader object at 0x7fb780512790>
nose2.session: DEBUG: Register method moduleLoadedSuite for plugin <nose2.plugins.loader.loadtests.LoadTestsLoader object at 0x7fb780512790>
nose2.session: DEBUG: Register method handleDir for plugin <nose2.plugins.loader.loadtests.LoadTestsLoader object at 0x7fb780512790>
nose2.session: DEBUG: Plugin is available: <class 'nose2.plugins.loader.parameters.Parameters'>
nose2.session: DEBUG: Register active plugin <nose2.plugins.loader.parameters.Parameters object at 0x7fb780512a50>
nose2.session: DEBUG: Register method loadTestsFromModule for plugin <nose2.plugins.loader.parameters.Parameters object at 0x7fb780512a50>
nose2.session: DEBUG: Register method loadTestsFromName for plugin <nose2.plugins.loader.parameters.Parameters object at 0x7fb780512a50>
nose2.session: DEBUG: Register method getTestCaseNames for plugin <nose2.plugins.loader.parameters.Parameters object at 0x7fb780512a50>
nose2.session: DEBUG: Register method getTestMethodNames for plugin <nose2.plugins.loader.parameters.Parameters object at 0x7fb780512a50>
nose2.session: DEBUG: Plugin is available: <class 'nose2.events.Plugin'>
nose2.session: DEBUG: Plugin is available: <class 'nose2.plugins.debugger.Debugger'>
nose2.session: DEBUG: Plugin is available: <class 'nose2.plugins.dundertest.DunderTestFilter'>
nose2.session: DEBUG: Register active plugin <nose2.plugins.dundertest.DunderTestFilter object at 0x7fb780516110>
nose2.session: DEBUG: Register method startTestRun for plugin <nose2.plugins.dundertest.DunderTestFilter object at 0x7fb780516110>
nose2.session: DEBUG: Register active plugin <nose2.plugins.coverage.Coverage object at 0x7fb78050ee90>
nose2.session: DEBUG: Register method startTestRun for plugin <nose2.plugins.coverage.Coverage object at 0x7fb78050ee90>
nose2.session: DEBUG: Register method afterSummaryReport for plugin <nose2.plugins.coverage.Coverage object at 0x7fb78050ee90>
nose2.main: DEBUG: Create tests from []/None
nose2.plugins.loader.discovery: DEBUG: Received event LoadFromNames(names=[], module=None)
nose2.plugins.loader.discovery: DEBUG: _discover in /var/tmp/portage/dev-python/nose2-0.5.0/work/nose2-0.5.0/nose2/tests/functional/support/scenario/test_with_module (/var/tmp/portage/dev-python/nose2-0.5.0/work/nose2-0.5.0/nose2/tests/functional/support/scenario/test_with_module)
nose2.plugins.loader.discovery: DEBUG: _find_tests('/var/tmp/portage/dev-python/nose2-0.5.0/work/nose2-0.5.0/nose2/tests/functional/support/scenario/test_with_module', '/var/tmp/portage/dev-python/nose2-0.5.0/work/nose2-0.5.0/nose2/tests/functional/support/scenario/test_with_module')
nose2.plugins.loader.discovery: DEBUG: find in dir /var/tmp/portage/dev-python/nose2-0.5.0/work/nose2-0.5.0/nose2/tests/functional/support/scenario/test_with_module (/var/tmp/portage/dev-python/nose2-0.5.0/work/nose2-0.5.0/nose2/tests/functional/support/scenario/test_with_module)
nose2.plugins.loader.discovery: DEBUG: _find_tests('/var/tmp/portage/dev-python/nose2-0.5.0/work/nose2-0.5.0/nose2/tests/functional/support/scenario/test_with_module/lib', '/var/tmp/portage/dev-python/nose2-0.5.0/work/nose2-0.5.0/nose2/tests/functional/support/scenario/test_with_module')
nose2.plugins.loader.discovery: DEBUG: find in dir /var/tmp/portage/dev-python/nose2-0.5.0/work/nose2-0.5.0/nose2/tests/functional/support/scenario/test_with_module/lib (/var/tmp/portage/dev-python/nose2-0.5.0/work/nose2-0.5.0/nose2/tests/functional/support/scenario/test_with_module)
nose2.plugins.loader.discovery: DEBUG: find in file /var/tmp/portage/dev-python/nose2-0.5.0/work/nose2-0.5.0/nose2/tests/functional/support/scenario/test_with_module/lib/__init__.py (/var/tmp/portage/dev-python/nose2-0.5.0/work/nose2-0.5.0/nose2/tests/functional/support/scenario/test_with_module)
nose2.plugins.loader.discovery: DEBUG: find in file /var/tmp/portage/dev-python/nose2-0.5.0/work/nose2-0.5.0/nose2/tests/functional/support/scenario/test_with_module/lib/mod1.py (/var/tmp/portage/dev-python/nose2-0.5.0/work/nose2-0.5.0/nose2/tests/functional/support/scenario/test_with_module)
nose2.plugins.loader.discovery: DEBUG: find in file /var/tmp/portage/dev-python/nose2-0.5.0/work/nose2-0.5.0/nose2/tests/functional/support/scenario/test_with_module/test_coverage.py (/var/tmp/portage/dev-python/nose2-0.5.0/work/nose2-0.5.0/nose2/tests/functional/support/scenario/test_with_module)
nose2.plugins.loader.generators: DEBUG: loadTestsFromTestCase <class 'test_coverage.TestLib'>
nose2.plugins.loader.generators: DEBUG: getTestCaseNames <class 'test_coverage.TestLib'>
nose2.plugins.loader.parameters: DEBUG: getTestCaseNames GetTestCaseNamesEvent(handled=False, loader=<PluggableTestLoader>, testCase=<class 'test_coverage.TestLib'>, testMethodPrefix=None, extraNames=[], excludedNames=[], isTestMethod=<function TestCaseLoader._getTestCaseNames.<locals>.isTestMethod at 0x7fb780e8f560>)
nose2.loader: DEBUG: loadTestsFromNames event LoadFromNames(names=[], module=None) result <unittest.suite.TestSuite tests=[<unittest.suite.TestSuite tests=[<unittest.suite.TestSuite tests=[<test_coverage.TestLib testMethod=test1>]>]>]>
--------------------- >> end captured logging << ---------------------
----------------------------------------------------------------------
```

--- a/.travis.yml
+++ b/.travis.yml
@@ -9,10 +9,10 @@ python:
   - "pypy3"
 install:
   - travis_retry pip install -r requirements-${TRAVIS_PYTHON_VERSION}.txt --allow-external argparse
-  - travis_retry pip install coveralls
-  - travis_retry pip install coverage
+  - if [[ "${TRAVIS_PYTHON_VERSION}" != 3.2 ]]; then travis_retry pip install coveralls; fi
+  - if [[ "${TRAVIS_PYTHON_VERSION}" != 3.2 ]]; then travis_retry pip install coverage; fi
 script:
-  - PYTHONPATH=. coverage run -m nose2.__main__ -v
+  - if [[ "${TRAVIS_PYTHON_VERSION}" != 3.2 ]]; then CMD="coverage run"; else CMD="python"; fi; PYTHONPATH=. $CMD -m nose2.__main__ -v
 after_success:
   - coveralls
 deploy:

--- a/nose2/tests/functional/test_coverage.py
+++ b/nose2/tests/functional/test_coverage.py
@@ -16,7 +16,7 @@ class TestCoverage(FunctionalTestCase):
         stdout, stderr = proc.communicate()
         self.assertTestRunOutputMatches(
             proc,
-            stderr=os.path.join('lib', 'mod1').replace('\\', r'\\') + STATS)
+            stderr=os.path.join('lib', 'mod1.py').replace('\\', r'\\') + STATS)
         self.assertTestRunOutputMatches(
             proc,
-            stderr='TOTAL   ' + STATS)
+            stderr='TOTAL      ' + STATS)

--- a/nose2/tests/functional/test_coverage.py
+++ b/nose2/tests/functional/test_coverage.py
@@ -1,9 +1,14 @@
 import os.path
+import platform
+from nose2.compat import unittest
 
 from nose2.tests._common import FunctionalTestCase
 
 
 class TestCoverage(FunctionalTestCase):
+    @unittest.skipIf(
+        platform.python_version_tuple()[:2] == ('3', '2'),
+        'coverage package does not support python 3.2')
     def test_run(self):
         proc = self.runIn(
             'scenario/test_with_module',

